lib/api/dsl/storage/index.js

```diff
@@ -529,7 +547,7 @@ function reindexTestTable(filters, subfilters, testTables, index, collection) {
   }
 
   // rebuild conditions index
-  conditions = new Uint8Array(tt.conditions.length - tt.removedConditions.array.length);
+  const conditions = new Uint8Array(tt.conditions.length - tt.removedConditions.array.length);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Split this 94 characters long line (which is greater than 80 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ALineLength)